modules/hcp-bm-hc.adoc

```diff
@@ -58,6 +59,7 @@ $ hcp create cluster agent \
 <8> Specify your hosted cluster namespace.
 <9> Specify the availability policy for the hosted control plane components. Supported options are `SingleReplica` and `HighlyAvailable`. The default value is `HighlyAvailable`.
 <10> Specify the supported {product-title} version that you want to use, for example, `4.17.0-multi`. If you are using a disconnected environment, replace `<ocp_release_image>` with the digest image. To extract the {product-title} release image digest, see _Extracting the {product-title} release image digest_.
+<11> Specify the node pool replica count, for example, `3`. You must specify the replica count as `0` or greater, to create the same number of replicas. Otherwise, no node pools are created.
```

Comment:
```suggestion
<11> Specify the node pool replica count, for example, `3`. You must specify the replica count as `0` or greater to create the same number of replicas. Otherwise, no node pools are created.
```